### PR TITLE
Issue #4: Add .circleci

### PR DESCRIPTION
**Pull request by Rahulkhinchi03**
Adding Circle CI to the website repository to automate some tasks.

### CODE REVIEW

**Comment by Rahulkhinchi03** [comment]
For testing, I just created a temporary Hello World job to run, soon we will add other checks as we start.

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    # See: https://circleci.com/docs/2.0/configuration-reference/#docker-machine-macos-windows-executor
+    docker:
+      - image: cimg/base:stable
+    # Add steps to the job
+    # See: https://circleci.com/docs/2.0/configuration-reference/#steps
+    steps:
+      - checkout
+      - run:
+          name: "Say hello"
+          command: "echo Hello, World!"
```
Comment (AshAman999) [inline .circleci/config.yml:19]: Not an expert in DevOps but shouldn't it be `echo "hello world!"`

**Review by AshAman999 — COMMENTED** [review]
@Rahulkhinchi03 , I guess the bash command might be kinda wrong, 
Not sure though but better if you could have a look

File: .circleci/config.yml (hunk 1 above)
Comment (Rahulkhinchi03) [reply .circleci/config.yml:19]: @AshAman999 We write everything in `""` for command.

You can check the execution here: https://app.circleci.com/pipelines/github/DeCodeCafe/Community-Website/1/workflows/b231edde-eeac-4d07-b307-992bce5b4152/jobs/1

Comment (AshAman999) [reply .circleci/config.yml:19]: Okay then, resolving and approving

**Review by AshAman999 — APPROVED** [review]

**Comment by Rahulkhinchi03** [comment]
@Vyom-Yadav I am merging this PR. We can have more issues once we have something in our files. So that we can have more engagement to change something.

**Pull request merged**